webapp/lib/index.ts

```diff
@@ -0,0 +1,5 @@
+import FlameGraphRenderer from '../javascript/components/FlameGraph/FlameGraphRenderer';
+import Flamegraph from '../javascript/components/FlameGraph/FlameGraphComponent/Flamegraph';
+import ExportData from '../javascript/components/ExportData';
+
+export { FlameGraphRenderer as FlamegraphRenderer, Flamegraph, ExportData };
```

Comment:
I think we should also add this one:
```
import { decodeFlamebearer } from '../node_modules/pyroscope/webapp/javascript/models/flamebearer';
```

Because my understanding is that if the data is coming from a server you need this function to convert it into something the component understands, right?

Reply:
That's correct, will do.

However just so that we are on the same page, the plan is to move that `decodeFlamebearer` function to happen within the `FlamegraphRenderer` component, so that the API is impler.